scripts/generate_multiple_forecasts.py

```diff
@@ -0,0 +1,68 @@
+"""Make a script to generate forecasts for multiple sites.
+
+The idea is to generate forecasts for multiple sites and to combine them into a 
+single csv. A list with several sites containing pv_id, latitude, longitude 
+and capacity will be passed through the forecaster then each forecast will be saved 
+to a csv.
+
+"""
+
+import pandas as pd
+from quartz_solar_forecast.forecast import run_forecast
+from quartz_solar_forecast.pydantic_models import PVSite
+from datetime import datetime
+
+def generate_forecasts(sites):
+    """Generate forecasts for pv_sites."""
+    csv_data = []
+
+    # run forecast for each pv site
+    for pv_id, site in sites.items():
+        predictions_df = run_forecast(site=site)
+
+        # add to csv data
+        for index, row in predictions_df.iterrows():
+            csv_row = [
+                pv_id,
+                site.latitude,
+                site.longitude,
+                site.capacity_kwp,
+                datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
+                index.date(),
+                index.time(),
+                row["power_wh"],
+            ]
+            csv_data.append(csv_row)
+    
+        csv_df = pd.DataFrame(
+            csv_data,
+            columns=[
+                "pv_id",
+                "latitude",
+                "longitude",
+                "capacity_kwp",
+                "forecast creation time",
+                "forecast date",
+                "forecast time",
+                "power_wh",
+            ],
+        )
+        
+
+        # write to a new csv
+        csv_df.to_csv(
```

Comment:
Maybe take this csv saving outside of the for loop as its saving multiple copies of the same thing

Reply:
and call it something like, "forecast_multiple_sites_{INSERT FORECAST CREATION TIME}.csv

Reply:
please include the forecast creation time in the csv name